### PR TITLE
remove hard coded yml file, update steps, add links 

### PR DESCRIPTION
**Pull request by mrjones-plip**
per #362

### CODE REVIEW

**Comment by mrjones-plip** [comment]
FYI @garethbowen  and @smbuthia

File: content/en/apps/tutorials/local-setup.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ relatedContent: >
 ---
 
 {{% pageinfo %}}
-This tutorial will take you through setting up a local environment to build and test CHT applications. This includes setting up the necessary tools to download and run the CHT public docker image as well as a command line interface tool to manage and build CHT apps.
+This tutorial will take you through setting up a local environment to build and test CHT applications on CHT version 3.9.1. This includes setting up the necessary tools to download and run the CHT public docker image as well as a command line interface tool to manage and build CHT apps.
```
Comment (newtewt) [inline content/en/apps/tutorials/local-setup.md:13]: Wish we had a way to make this dynamic on the version number.  It's a manual maintenance item and in docs sites that usually means it gets forgotten.

File: content/en/core/guides/docker-setup.md [hunk 2]
```diff
@@ -35,47 +35,10 @@ CPUs: 2
 
 ## Use Docker-Compose:
 
-In the location you would like to host your configuration files, create a file titled <project_name>-medic-os-compose.yml with the following contents:
+In the location you would like to host your configuration files, create a file titled <project_name>-medic-os-compose.yml with the following contents. In this example we're creating a file called `test-docs-medic-os-compose.yml` via `curl`:
```
Comment (newtewt) [inline content/en/core/guides/docker-setup.md:39]: My only thought is that not every OS you can run docker on has curl. Might add an href and a note about right clicking to save as but it's up to you really.  Also, maybe a ticket for another day. We specify a name for the file. If the assumption is the reader is first time using docker-compose they likely won't have any other compose files. 

`<a href="https://raw.githubusercontent.com/medic/cht-core/master/docker-compose.yml" >Right Click to Save the docker file. </a> `

**Review by newtewt — APPROVED** [review]

File: content/en/core/guides/docker-setup.md (hunk 2 above)
Comment (mrjones-plip) [reply content/en/core/guides/docker-setup.md:39]: yeah - this is totally worth an update.  Same for the [local set up page](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/) too (deep link to raw yml).  Stay tuned

File: content/en/apps/tutorials/local-setup.md (hunk 1 above)
Comment (mrjones-plip) [reply content/en/apps/tutorials/local-setup.md:13]: There's [custom variables in hugo](https://www.mikedane.com/static-site-generators/hugo/variables/) which we could try and leverage. However, maybe when ever we update the docker instance we have a mini release process that updates this text?  Currently it looks to be only in one place.  Alternately, we could say something along the lines of "See our [Supported Versions](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) page" and then denote which version the docker image is on and _this_ to the docker image release process?  That way we'd only have to reference the one `supported-software` page.

 Also, maybe we solve it when we solve when [we unify our install docs](https://github.com/medic/cht-docs/issues/317)?

File: content/en/core/guides/docker-setup.md (hunk 2 above)
Comment (mrjones-plip) [reply content/en/core/guides/docker-setup.md:41]: Done! see 3cc65ec

Comment (newtewt) [reply content/en/core/guides/docker-setup.md:41]: solid!

File: content/en/apps/tutorials/local-setup.md (hunk 1 above)
Comment (newtewt) [reply content/en/apps/tutorials/local-setup.md:13]: works for me.

**Review by newtewt — APPROVED** [review]

**Pull request merged**